packages/base-styles/_mixins.scss

```diff
@@ -437,3 +437,53 @@
 		max-width: 100%;
 	}
 }
+
+@mixin admin-scheme($color-primary, $color-secondary, $color-toggle) {
+	--wp-admin-theme-color: #{$color-primary};
+	--wp-admin-theme-color-secondary: #{$color-secondary};
+	--wp-admin-theme-color-toggle: #{$color-toggle};
+
+	// Ideally, we should limit these computed variables,
+	// because we can"t compute colors based on CSS variables.
+	// This requires refactoring some existing styles.
+	--wp-admin-theme-color-darker-20: darken(#{$color-primary}, 20%);
+	--wp-admin-theme-color-darker-10: darken(#{$color-primary}, 10%);
+	--wp-admin-theme-color-lighter-10: lighten(#{$color-primary}, 10%);
+	--wp-admin-theme-color-lighter-40: lighten(#{$color-primary}, 40%);
```

Comment:
The `darken` & `lighten` commands aren't being applied in sass, because sass [passes through all values of custom properties.](https://sass-lang.com/documentation/style-rules/declarations#custom-properties) You'll want to wrap these in `#{}`, ex `#{darken($color-primary, 20%)};`. You can remove the interpolation of `$color-primary` in the function if it's a color (not quoted).